src/main/java/net/minecraftforge/fluids/Fluid.java

```diff
@@ -126,10 +124,15 @@
      */
     protected Block block = null;
 
+    //TODO Remove in 1.13
+    @Deprecated //Use constructor below
```

Comment:
That's a really good point, it's worth removing it now since everyone will have to fix their fluid code anyway.